Extensions/EdFi.Ods.Profiles.Sample/assemblyMetadata.json

```diff
@@ -0,0 +1,4 @@
+{
+  "assemblyModelType": "profile",
+  "assemblyMetadataFormatVersion": "1.0.0"
```

Comment:
On the `AssemblyMetadata` class added on the PR for ODS-4664, you should probably add the `assemblyMetadataFormatVersion` property as well.

It also makes me think that you should probably add it to the other JSON metadata files in the existing extensions projects.